evm/.example.hardhat.config.js

```diff
@@ -3,7 +3,7 @@ require('@nomicfoundation/hardhat-toolbox');
 
 const env = process.env.ENV || 'testnet';
 const { importNetworks, readJSON } = require('@axelar-network/axelar-contract-deployments');
-const chains = require(`@axelar-network/axelar-contract-deployments/info/${env}.json`);
+const chains = require(`@axelar-network/axelar-contract-deployments/axelar-chains-info/info/${env}.json`);
 const keys = readJSON(`${__dirname}/info/keys.json`);
 const { networks, etherscan } = importNetworks(chains, keys);
```

Comment:
can you add readJSON, importNetworks in a utils.js that's exposed from this package as well? Avoids having to import contract deployments folder above
```suggestion
const keys = readJSON(`${__dirname}/axelar-contracts-config/info/keys.json`);
const { networks, etherscan } = importNetworks(chains, keys);
```